include/ansi_code_markup.h

```diff
@@ -31,4 +31,12 @@
  */
 std::string convert_ansi_markup(const char* str);
 
+/*!
+ * \brief Convert marked up strings to strings with ANSI codes
+ * 
+ * \param str 
+ * \return std::string 
+ */
+std::string convert_ansi_markup(std::string &str);
+
```

Comment:
This might have slipped in merging from (instead of rebasing against) `main`?

Reply:
No, I just thought it's handy to expose the std::string variant as well.